lib/Remotery.c

```diff
@@ -2313,10 +2347,10 @@ static void ObjectAllocator_Destructor(ObjectAllocator* allocator)
     // Destroy all objects released to the allocator
     while (allocator->first_free != NULL)
     {
-        ObjectLink* next = allocator->first_free->next;
+        ObjectLink* next = ((ObjectLink*)allocator->first_free)->next;
```

Comment:
the `std::atomic` doesn't have a `->` operator, but a [cast operator](https://en.cppreference.com/w/cpp/atomic/atomic/operator_T).

Reply:
Ideally we'd have a weak load here but we don't have a primitive for that.